docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``ice_concentration_threshold``| Value of sea ice concentration above which albedo should be set to ``ice_albedo_value``.                    | ``0.5``              |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``ice_file_name``              | Name of file containing sea ice concentration.                                                              | ``siconc_clim_amip`` |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``do_ape_sst``                 | Flag to set prescribed SST according to the APE aquaplanet analytic form                                    | ``False``            |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+|``add_latent_heat_flux_anom``  | Flag to add an anomalous latent heat flux                                                                   | ``False``            |
++-------------------------------+-------------------------------------------------------------------------------------------------------------+----------------------+
+
+Diagnostics
+-------------------
++-------------------+-------------------------------------+------------------------------------+
+| Name              | Description                         | Units                              |
++===================+=====================================+====================================+
+| t_surf            | Surface temperature                 | :math:`\text{K}\`                  |
++-------------------+-------------------------------------+------------------------------------+
+| delta_t_surf      | Surface temperature change          | :math:`\text{K}\`                  |
++-------------------+-------------------------------------+------------------------------------+
+| flux_t            | Surface sensible heat flux          | :math:`\text{W}\,\text{m}^{-2}`    |
++-------------------+-------------------------------------+------------------------------------+
+| flux_lhe          | Surface latent heat flux            | :math:`\text{W}\,\text{m}^{-2}`    |
++-------------------+-------------------------------------+------------------------------------+
+| flux_oceanq       | Ocean heat flux                     | :math:`\text{W}\,\text{m}^{-2}`    |
++-------------------+-------------------------------------+------------------------------------+
+| ice_conc          | Sea ice concentration               | None                               |
```

Comment:
Perhaps rather than saying none in these two, say Unitless with a range of 0-1 or similar?